src/shared/logger/activation.ts

```diff
@@ -159,4 +164,29 @@ async function registerLoggerCommands(context: vscode.ExtensionContext): Promise
             recordVscodeViewLogs()
         })
     )
+    context.subscriptions.push(
+        vscode.commands.registerCommand('aws.viewLogsAtMessage', async (msgPromise: Promise<string | undefined>) => {
```

Comment:
I would probably put this in a named function, that  makes the stacktrace more useful if anything goes wrong.